BaragonCore/src/main/java/com/hubspot/baragon/models/RequestAction.java

```diff
@@ -2,8 +2,10 @@
 
 public enum RequestAction {
   UPDATE,
+  UPDATE_AND_PURGE_CACHE,
```

Comment:
@ssalinas What are your thoughts on this? I played around with adding a `purgeRequest` field to a `BaragonRequest` but that felt a bit awkward given that there was already the action field set to `PURGE_CACHE`. I'm not sure this is less awkward, but I can say it was much easier to wire up.

Reply:
This is where I meant to make that comment, seems it's messing up older clients since they don't have the new enum value. Trying to think of a good way to add this

Reply:
What if we drop `UPDATE_AND_PURGE_CACHE` and add the call to purgeCache() in the normal UPDATE flow?